src/interface/TWCoinTypeConfiguration.cpp

```diff
@@ -169,6 +172,7 @@ TWString *_Nullable TWCoinTypeConfigurationGetTransactionURL(enum TWCoinType typ
         break;
     case TWCoinTypeLux:
         url += "/#/tx/" + txId;
+    case TWCoinTypeBravo: break;
```

Comment:
we should construct a tx url here

Reply:
we do not have such url atm. https://explorer.bravocoin.com/

Reply:
How can a user view coin transfer tx/action?

Reply:
It has been implemented in explorer. Added URL.